src/cpufreq.c

```diff
@@ -18,6 +18,9 @@
  *
  * Authors:
  *   Peter Holik <peter at holik.at>
+ *
+ * Changes:
+ *   2016-05-27: freq_fname_rhel was added by Igor Solovyov <igor dot solovyov at gmail dot com>.
```

Comment:
Nit: You can just add yourself to Authors. The details of what you did can be extracted from the Git history.